web-components/src/[sandbox]/sandbox.ts

```diff
@@ -103,13 +104,36 @@ export class Sandbox extends LitElement {
         </label>
         <label class="switch">
           <input
-            type="checkbox"
+            type="radio"
+            name="theme-switch"
+            class="momentum-switch"
+            data-aspect="momentum"
+            @click=${this.toggleSetting}
+            ?checked=${this.theme === "momentum"}
+          />
+          Momentum
+        </label>
+        <label class="switch">
+          <input
+            type="radio"
+            name="theme-switch"
             class="lumos-switch"
             data-aspect="lumos"
             @click=${this.toggleSetting}
-            ?checked=${this.lumos}
+            ?checked=${this.theme === "lumos"}
+          />
+          Lumos
+        </label>
+        <label class="switch">
+          <input
+            type="radio"
+            name="theme-switch"
+            class="momentumv2-switch"
+            data-aspect="momentumV2"
+            @click=${this.toggleSetting}
+            ?checked=${this.theme === "momentumV2"}
           />
-          Lumos Theme
+          MomentumV2
         </label>
```

Comment:
Update sandbox to radio buttons for theme instead of checkbox now that there are multiple themes